include/revolution/os/OSFont.h

```diff
@@ -49,7 +49,7 @@ u32 OSLoadFont(OSFontHeader* font, void* dst);
 const char* OSGetFontTexel(const char* str, void* dst, s32 xOfs, s32 arg3, u32* widthOut);
 bool OSInitFont(OSFontHeader* font);
 const char* OSGetFontTexture(const char* str, void** texOut, u32* xOut, u32* yOut, u32* widthOut);
-const char* OSGetFontWidth(const char* str, u32* widthOut);
+char* OSGetFontWidth(const char* str, u32* widthOut);
```

Comment:
I don't think we should be removing `const` here. Can it be fixed by making some other things `const` wherever it's used?

Reply:
hmm, I think ideally we'd try to be `const`-correct everywhere but that could go pretty deep, so seems good enough for now thanks